image_provisioner/playbooks/roles/seal-image/tasks/main.yaml

```diff
@@ -3,6 +3,14 @@
   shell:  yum clean all
   ignore_errors: true
 
+- name: remove /root/.kube dir
+  shell: /bin/rm -rf /root/.kube
```

Comment:
Probably preferable to use the file task with state=empty.   Future enhancement.